README.md

```diff
@@ -34,3 +34,5 @@ pools.sv_used | pgbouncer_pools_server_used_connections | Server connections idl
 pools.sv_tested | pgbouncer_pools_server_testing_connections | Server connections currently running either server_reset_query or server_check_query, shown as connection
 pools.sv_login | pgbouncer_pools_server_login_connections | Server connections currently in the process of logging in, shown as connection
 pools.maxwait | pgbouncer_pools_client_maxwait_seconds | Age of oldest unserved client connection, shown as second
+config.max_client_conn | pgbouncer_config_max_client_conn | Configured maximum number of client connections
```

Comment:
See https://prometheus.io/docs/practices/naming/.

@SuperQ Should this be `pgbouncer_config_max_client_conns` or `pgbouncer_config_max_client_connections`?

Reply:
@stanhu @SuperQ, thank you for the input so far, we really appreciate it - is there anything we can do to help this move forward? Please let us know

Reply:
Thanks for your feedback. We will update the PR accordingly.